servers/audio/audio_filter_sw.h

```diff
@@ -36,11 +36,11 @@
 class AudioFilterSW {
 public:
 	struct Coeffs {
-		float a1 = 0.0f;
-		float a2 = 0.0f;
-		float b0 = 0.0f;
-		float b1 = 0.0f;
-		float b2 = 0.0f;
+		double a1 = 0.0f;
+		double a2 = 0.0f;
+		double b0 = 0.0f;
+		double b1 = 0.0f;
+		double b2 = 0.0f;
```

Comment:
Should use double literals too:
```suggestion
		double a1 = 0.0;
		double a2 = 0.0;
		double b0 = 0.0;
		double b1 = 0.0;
		double b2 = 0.0;
```